services/blockchain-connector/tests/functional/formatter.test.js

```diff
@@ -49,69 +48,70 @@ describe('Functional tests for decoder', () => {
 	afterAll(() => broker.stop());
 
 	it('decode Transaction', async () => {
```

Comment:
Can we please adjust all the test descriptions to be relevant as well?

Reply:
Applied.